tests/devices/qutrit_mixed/test_qutrit_mixed_simulate.py

```diff
@@ -0,0 +1,735 @@
+# Copyright 2018-2024 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Unit tests for simulate in devices/qutrit_mixed."""
+
+import pytest
+import numpy as np
+import pennylane as qml
+from pennylane import math
+
+from pennylane.devices.qutrit_mixed import simulate, get_final_state, measure_final_state
+
+
+def expected_TRX_circ_expval_values(phi, subspace):
+    """Find the expect-values of GellManns 2,3,5,8
+    on a circuit with a TRX gate"""
+    if subspace == (0, 1):
+        gellmann_2 = -np.sin(phi)
+        gellmann_3 = np.cos(phi)
+        gellmann_5 = 0
+        gellmann_8 = np.sqrt(1 / 3)
+    if subspace == (0, 2):
+        gellmann_2 = 0
+        gellmann_3 = np.cos(phi / 2) ** 2
+        gellmann_5 = -np.sin(phi)
+        gellmann_8 = np.sqrt(1 / 3) * (np.cos(phi) - np.sin(phi / 2) ** 2)
+    return np.array([gellmann_2, gellmann_3, gellmann_5, gellmann_8])
+
+
+def expected_TRX_circ_expval_jacobians(phi, subspace):
+    """Find the Jacobians of expect-values of GellManns 2,3,5,8
+    on a circuit with a TRX gate"""
+    if subspace == (0, 1):
+        gellmann_2 = -np.cos(phi)
+        gellmann_3 = -np.sin(phi)
+        gellmann_5 = 0
+        gellmann_8 = 0
+    if subspace == (0, 2):
+        gellmann_2 = 0
+        gellmann_3 = -np.sin(phi) / 2
+        gellmann_5 = -np.cos(phi)
+        gellmann_8 = np.sqrt(1 / 3) * -(1.5 * np.sin(phi))
+    return np.array([gellmann_2, gellmann_3, gellmann_5, gellmann_8])
+
+
+def expected_TRX_circ_state(phi, subspace):
+    """Find the state after applying TRX gate on |0>"""
+    expected_vector = np.array([0, 0, 0], dtype=complex)
+    expected_vector[subspace[0]] = np.cos(phi / 2)
+    expected_vector[subspace[1]] = -1j * np.sin(phi / 2)
+    return np.outer(expected_vector, np.conj(expected_vector))
+
+
+class TestCurrentlyUnsupportedCases:
+    """Test currently unsupported cases, such as sampling expval or samples without shots"""
```

Comment:
Why are these two cases currently unsupported?

Reply:
This was from the qubit tests, I don't know in the future if there is a planned change but currently you can't take sample without shots, also currently on this device you can't take probs with shots